.github/workflows/release.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Publish Release'
+on:
+  release:
+    types: [published]
+jobs:
+  sbt_release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: guardian/actions-sbt-release@v3
+        with:
+            fetchDepth: 0
```

Comment:
At one point I did have this in the fezziwig build, but actually I don't think it's needed normally: only when the tagged build is really deep in commit history.

Reply:
I guess we can remove it and add it in the future if needed